horovod/torch/CMakeLists.txt

```diff
@@ -16,13 +16,8 @@ endif()
 
 if (HAVE_CUDA AND NOT Pytorch_CUDA)
     message(FATAL_ERROR "Horovod build with GPU support was requested but this PyTorch installation does not support CUDA.")
-elseif (Pytorch_CUDA AND NOT HAVE_CUDA)
-    add_cuda()
-endif()
-if (HAVE_ROCM AND NOT Pytorch_ROCM)
+elseif (HAVE_ROCM AND NOT Pytorch_ROCM)
```

Comment:
This was causing cmake to fail because for some reason add_cuda() was getting called and I didn't dig into why.  Now that I've looked at it more closely, we should change FindPytorch.cmake to better detect cuda vs rocm.  Let me do that.

Reply:
Fixed by d2195d22e75ca9e968e2c80942f4b9ad72582189.